### PR TITLE
vowpal-wabbit 9.6.0

### PR DESCRIPTION
**Pull request by byronxu99**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/vowpal-wabbit.rb
Comment (byronxu99) [reply Formula/vowpal-wabbit.rb:25]: Yes, we have the same change being made here: https://github.com/VowpalWabbit/vowpal_wabbit/pull/4275

Comment (SMillerDev) [reply Formula/vowpal-wabbit.rb:25]: Could we wait until that's merged and use the upstream commit then?

Comment (byronxu99) [reply Formula/vowpal-wabbit.rb:25]: We are releasing our official 9.6.0 version without the change, as it only seems to affect building through Homebrew. To keep the code itself consistent across all platforms, we'd prefer to use the patch for Homebrew here.

Comment (SMillerDev) [reply Formula/vowpal-wabbit.rb:25]: Only patching Homebrew is okay, I'm just requesting you use the bellow format for the patch instead of `patch DATA`:
```rb
patch do
  url "the url to the merged upstream commit in patch format"
  sha256 ""
end
```

Comment (byronxu99) [reply Formula/vowpal-wabbit.rb]: The patch has been updated now.

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**
